ext/standard/credits_ext.h

```diff
@@ -1,7 +1,7 @@
 /*
                       DO NOT EDIT THIS FILE!
 
- it has been automaticaly created by php7/scripts/credits from
+ it has been automatically created by php7/scripts/credits from
```

Comment:
Probably not necessary to edit this file, since it's autogenerated (same with ext/standard/credits_sapi.h).

Reply:
I also edit the script at `scripts/dev/credits` to avoid overwriting these changes.